pipelines/skylab/smartseq2_multisample/mouse_paired_example.json

```diff
@@ -1,12 +1,12 @@
 {
-    "MultiSampleSmartSeq2.genome_ref_fasta": "gs://hca-dcp-analysis-pipelines-reference/alignmentReferences/GRCm38_Gencode_vM21/GRCm38.primary_assembly.genome.fa",
-    "MultiSampleSmartSeq2.rrna_intervals": "gs://hca-dcp-analysis-pipelines-reference/alignmentReferences/GRCm38_Gencode_vM21/gencode.vM21.primary_assembly.rRNA.interval_list",
-    "MultiSampleSmartSeq2.gene_ref_flat": "gs://hca-dcp-analysis-pipelines-reference/alignmentReferences/GRCm38_Gencode_vM21/gencode.vM21.primary_assembly.refflat.txt",
+    "MultiSampleSmartSeq2.genome_ref_fasta": "gs://hca-dcp-sc-pipelines-test-data/alignmentReferences/20210309/mm10/GRCm38.primary_assembly.genome.fa",
+    "MultiSampleSmartSeq2.rrna_intervals": "gs://hca-dcp-sc-pipelines-test-data/alignmentReferences/20210309/mm10/gencode.vM21.primary_assembly.annotation.interval_list",
+    "MultiSampleSmartSeq2.gene_ref_flat": "gs://hca-dcp-sc-pipelines-test-data/alignmentReferences/20210309/mm10/gencode.vM21.primary_assembly.annotation.refflat.txt",
     "MultiSampleSmartSeq2.hisat2_ref_name": "hisat2_primary_gencode_mouse_vM21",
     "MultiSampleSmartSeq2.hisat2_ref_trans_name": "hisat2_from_rsem_star_primary_gencode_mouse_vM21",
-    "MultiSampleSmartSeq2.hisat2_ref_index": "gs://hca-dcp-analysis-pipelines-reference/alignmentReferences/GRCm38_Gencode_vM21/hisat2_primary_gencode_mouse_vM21.tar.gz",
-    "MultiSampleSmartSeq2.hisat2_ref_trans_index": "gs://hca-dcp-analysis-pipelines-reference/alignmentReferences/GRCm38_Gencode_vM21/hisat2_from_rsem_star_primary_gencode_mouse_vM21.tar.gz",
-    "MultiSampleSmartSeq2.rsem_ref_index": "gs://hca-dcp-analysis-pipelines-reference/alignmentReferences/GRCm38_Gencode_vM21/rsem_primary_gencode_mouse_vM21.tar",
+    "MultiSampleSmartSeq2.hisat2_ref_index": "gs://hca-dcp-sc-pipelines-test-data/alignmentReferences/20210309/mm10/hisat2_primary_gencode_mouse_vM21.tar.gz",
+    "MultiSampleSmartSeq2.hisat2_ref_trans_index": "gs://hca-dcp-sc-pipelines-test-data/alignmentReferences/20210309/mm10/hisat2_from_rsem_star_primary_gencode_mouse_vM21.tar.gz",
+    "MultiSampleSmartSeq2.rsem_ref_index": "gs://hca-dcp-sc-pipelines-test-data/alignmentReferences/20210309/mm10/star_primary_gencode_mouse_vM21.tar",
```

Comment:
I assume the star and rsem references are equivalent? @kishorikonwar is that correct?